### DISCUSSION TITLE
Article: A simple REST API with Blacksheep and Piccolo

### DISCUSSION THREAD

**Discussion by dantownsend** [piccolo-orm/piccolo#542] [General]
A nice introduction to BlackSheep and Piccolo, by @carlosm27.

https://carlosmv.hashnode.dev/a-simple-rest-api-with-blacksheep-and-piccolo-orm-or-python

**Comment by sinisaos** [comment]
This is great. Thanks @carlosm27. I also played a bit with Blacksheep and Piccolo. Here is [repo](https://github.com/sinisaos/blacksheep-piccolo). @dantownsend Maybe you can add these two examples to [Piccolo examples](https://github.com/piccolo-orm/piccolo_examples) repo. It may be useful to someone.

**Reply by dantownsend** [reply]
Yeah, good idea - I've added them.

**Reply by sinisaos** [reply]
Great. Thanks.
